src/cloudai/workloads/nemo_run/cloudai_nemorun.py

```diff
@@ -832,19 +848,56 @@ def cloudai_llama3_405b_recipe() -> run.Partial:
     set_enable_cuda_graphs_params(recipe)
     recipe.trainer.strategy.cross_entropy_fusion_impl = "te"
 
+    gpu_type = os.getenv("CLOUDAI_GPU_TYPE")
```

Comment:
We have the identical block for llama70b function, let's merge them under a single function.